python/ql/lib/semmle/python/RegexTreeView.qll

```diff
@@ -1009,3 +1028,24 @@ class RegExpBackRef extends RegExpTerm, TRegExpBackRef {
 
 /** Gets the parse tree resulting from parsing `re`, if such has been constructed. */
 RegExpTerm getParsedRegExp(StrConst re) { result.getRegex() = re and result.isRootTerm() }
+
+/**
+ * A node whose value may flow to a position where it is interpreted
```

Comment:
@tausbn You asked for this class to me moved to another location. Could you tell me the preferred place?

Reply:
I would just create a file in `lib/semmle/python/dataflow/new` called `Regex.qll`, and import it appropriately. Would that be okay?